app/services/decorators/capital_summary_decorator.rb

```diff
@@ -0,0 +1,55 @@
+module Decorators
+  class CapitalSummaryDecorator
+    attr_reader :assessment
+
+    def initialize(record)
+      @record = record
+    end
+
+    def as_json # rubocop:disable Metrics/MethodLength, Metrics/AbcSize
```

Comment:
Could as_json be renamed? This is because rails already has a `.as_json` method. Maybe rename to `.summary` or .result